sources/assets/Xenko.Core.Assets/PackageSessionPublicHelper.cs

```diff
@@ -64,8 +69,10 @@ private static void CheckMSBuildToolset()
             // Check that we can create a project
             using (var projectCollection = new Microsoft.Build.Evaluation.ProjectCollection())
             {
-                if (projectCollection.GetToolset("15.0") == null)
-                    throw new InvalidOperationException("Could not find MSBuild toolset 15.0");
+                if (s_msBuildToolVersions.Select(ver => projectCollection.GetToolset(ver)).Any())
```

Comment:
Is this check correct? Seems like it should be checking if there are *not* any.

Reply:
I had the same after thought before I went to bed last night, just wasn't sure.

I will correct it soon.